backend/utils/database_group.go

```diff
@@ -77,7 +78,11 @@ func GetStatementsAndSchemaGroupsFromSchemaGroups(statement string, engine store
 			continue
 		}
 		for _, schemaGroup := range schemaGroups {
-			if strings.Contains(singleStatement.Text, schemaGroup.Placeholder) {
+			re, err := regexp.Compile(`\b` + schemaGroup.Placeholder + `\b`)
+			if err != nil {
+				return nil, nil, errors.Wrapf(err, "failed to compile regexp")
+			}
+			if re.MatchString(singleStatement.Text) {
```

Comment:
Other than table name, this may match any parts of statement, such as keyword, column name and index name, so that the statement could match incorrect schema group.

Reply:
True.